2.0/runtime/jessie/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM microsoft/dotnet:2.0-runtime
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    pdftk \
+    ghostscript \
+    imagemagick \
+    gsfonts

```

Comment:
you may add  ``` && rm -rf /var/lib/apt/lists/*``` in the end just to reduce a bit the image size, as recommended in [best-practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run)